pkg/galaxy/server.go

```diff
@@ -156,6 +158,24 @@ func (g *Galaxy) requestFunc(req *galaxyapi.PodRequest) (data []byte, err error)
 	return
 }
 
+func parsePorts(pod *corev1.Pod) []k8s.Port {
+	var ports []k8s.Port
+	for _, container := range pod.Spec.Containers {
+		for _, port := range container.Ports {
+			tmp := k8s.Port{
+				HostPort:      port.HostPort,
```

Comment:
There is a problem with this. After this PR we'll make port mapping for all hostport=0 containerPorts for all pods, this may not be necessary. People usually want their host=0 ports be exported via service/noteport instead of host ports. We'd better do random port mapping for pods that have a special annotation ?

Reply:
done